roles/system/templates/deployments/system-app-deployment.yaml

```diff
@@ -0,0 +1,450 @@
+                secretKeyRef:
+                  key: enable_starttls_auto
+                  name: system-smtp
+          ### secret/system-master-apicast
+            - name: APICAST_ACCESS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  key: ACCESS_TOKEN
+                  name: system-master-apicast
+          ### secret/zync
+            - name: ZYNC_AUTHENTICATION_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  key: ZYNC_AUTHENTICATION_TOKEN
+                  name: zync
+          ### secret/backend-redis
+            - name: BACKEND_REDIS_URL
+              valueFrom:
+                secretKeyRef:
+                  key: REDIS_STORAGE_URL
+                  name: backend-redis
+            - name: BACKEND_REDIS_SENTINEL_HOSTS
+              valueFrom:
+                secretKeyRef:
+                  key: REDIS_STORAGE_SENTINEL_HOSTS
+                  name: backend-redis
+            - name: BACKEND_REDIS_SENTINEL_ROLE
+              valueFrom:
+                secretKeyRef:
+                  key: REDIS_STORAGE_SENTINEL_ROLE
+                  name: backend-redis
+          ### secret/backend-listener
+            - name: APICAST_BACKEND_ROOT_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  key: service_endpoint
+                  name : system-backend-listener
+            - name: BACKEND_ROUTE
+              valueFrom:
+                secretKeyRef:
+                  key: service_endpoint
+                  name : system-backend-listener
+            - name: BACKEND_PUBLIC_URL
+              valueFrom:
+                secretKeyRef:
+                  key: route_endpoint
+                  name : system-backend-listener
+          ### secret/backend-internal-api
+            - name: CONFIG_INTERNAL_API_USER
+              valueFrom:
+                secretKeyRef:
+                  key: username
+                  name: backend-internal-api
+            - name: CONFIG_INTERNAL_API_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: backend-internal-api
+            ### secret/system-multitenant-assets-s3
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  key: AWS_ACCESS_KEY_ID
+                  name: system-multitenant-assets-s3
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: AWS_SECRET_ACCESS_KEY
+                  name: system-multitenant-assets-s3
+            - name: AWS_BUCKET
+              valueFrom:
+                secretKeyRef:
+                  key: AWS_BUCKET_NAME
+                  name: system-multitenant-assets-s3
+            - name: AWS_REGION
+              valueFrom:
+                secretKeyRef:
+                  key: AWS_BUCKET_REGION
+                  name: system-multitenant-assets-s3
```

Comment:
It is missing the podAntiffinity config, isn't it?

Reply:
Added in https://github.com/3scale/saas-operator/pull/43/commits/102f64f1a4091bf1c9a90a9331ca5eb759cf3034